helm/alfresco-content-services-community/templates/ingress-repository.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
     nginx.ingress.kubernetes.io/session-cookie-hash: "sha1"
     # Default file limit (1m) check, document(s) above this size will throw 413 (Request Entity Too Large) error
     nginx.ingress.kubernetes.io/proxy-body-size: {{ .Values.repository.ingress.maxUploadSize }}
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      location ~ ^(/alfresco/s/api/solr/.*)$ {return 403;}
```

Comment:
There are a few more ways (URLs) to call the webscripts, please see the [web.xml](https://github.com/Alfresco/acs-packaging/blob/master/war/src/main/webapp/WEB-INF/web.xml)
As far as I remember you can at least use `/alfresco/s/*`, `/alfresco/service/*`, `/alfresco/wcs/*` and `/alfresco/wcservice/*` to invoke the same webcript,

Also you might want to use `.*` instead of `alfresco` in the URL as it is done for Share.

I would like to include @tunaaksoy and @michaelsuzukisagi into this review as well. These APIs (/alfresco/s/api/solr) were protected by SSL cert in previous versions. If we are blocking them in Helm charts why do we need them at all?